better_profanity/profanity.py

```diff
@@ -67,6 +67,12 @@ def load_censor_words(custom_words=None):
     # The default wordlist takes ~5MB+ of memory
     CENSOR_WORDSET = all_censor_words
 
+def add_censor_words(custom_words=None):
```

Comment:
I think the argument `custom_words` should not have a default value.

It doesn't make much sense to me to have a `None` argument and an unnecessary function call.